Sources/Core/AWSClientRuntime/Signing/AWSSigV4Signer.swift

```diff
@@ -11,14 +11,15 @@ import Foundation
 
 public class AWSSigV4Signer {
```

Comment:
Since this type now handles `sigv4` and `sigv4a`, should it be renamed for clarity?

Reply:
It is still SigV4, we ever have V5, then we will add another type eventually.